app/docker/filters/filters.js

```diff
@@ -322,4 +322,8 @@ angular
       }
       return fullName.substring(0, versionIdx);
     };
+  })
+  .filter('containerportfilter', function () {
```

Comment:
sorry, but why the name change? this filter can be used anywhere we want unique values

Reply:
You mean to use `containerport`?

Reply:
yes. rename it with `uniqueBy` or `unique`